<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="db_tab">
      <attribute name="title">
       <string>База данных</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Сервер БД</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dbhost_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Название БД</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="dbname_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Пользователь</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="dbuser_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="dbpassword_lineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scaner_tab">
      <attribute name="title">
       <string>Сканер штрих-кода</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Порт</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="port_comboBox">
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM32</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Префикс (HEX)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="prefix_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Суфикс (HEX)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="suffix_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Штрих-код</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Префикс весового</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="barcodePrefix_lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>KKM</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Порт</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="printerPort_comboBox">
         <item>
          <property name="text">
           <string>COM1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM31</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>COM32</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Снять отчет с гашением</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Скорость порта</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="printerPortSpeed_comboBox">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600 </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>dbhost_lineEdit</tabstop>
  <tabstop>dbname_lineEdit</tabstop>
  <tabstop>dbuser_lineEdit</tabstop>
  <tabstop>dbpassword_lineEdit</tabstop>
  <tabstop>port_comboBox</tabstop>
  <tabstop>prefix_lineEdit</tabstop>
  <tabstop>suffix_lineEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
